docs/sites.yml

```
@@ -8022,3 +8022,13 @@
     - Blog
     - Education
   featured: false
+- title: Rockwong Blog
+  main_url: http://rockwong.com/
```

The `main_url` doesn't point to gatsby site, let's make sure we are linking to gatsby site:
```suggestion
  main_url: http://rockwong.com/blog/
```